addons/medical/functions/fnc_displayPatientInformation.sqf

```diff
@@ -145,6 +145,13 @@ if (_show) then {
             (_display displayCtrl (_availableSelections select _foreachIndex)) ctrlSetTextColor [_red, _green, _blue, 1.0];
         } foreach _selectionBloodLoss;
 
+        private _tourniquets = _target getVariable [QGVAR(tourniquets), [0,0,0,0,0,0]];
```

Comment:
This is just a copy of `ace_medical_menu_fnc_updateTourniquets`
Call that instead of having duplicate code.

Reply:
See latest commit as reference

Reply:
Well if a medical function needs it. Then it seems to make sense to be in medical.